lib/handlers/injector-sor.ts

```diff
@@ -233,17 +234,28 @@ export abstract class InjectorSOR<Router, QueryParams> extends Injector<
             sourceOfTruthPoolProvider: noCacheV3PoolProvider,
           })
 
-          const tokenFeeFetcher = new OnChainTokenFeeFetcher(chainId, provider)
           const tokenValidatorProvider = new TokenValidatorProvider(
             chainId,
             multicall2Provider,
             new NodeJSCache(new NodeCache({ stdTTL: 30000, useClones: false }))
           )
-          const tokenPropertiesProvider = new TokenPropertiesProvider(
-            chainId,
-            new NodeJSCache(new NodeCache({ stdTTL: 30000, useClones: false })),
-            tokenFeeFetcher
-          )
+          let tokenPropertiesProvider: TokenPropertiesProvider
+          // Gradually switch to graphql token fee fetcher - Start with 10%.
+          if (Math.random() <= 0.1) {
```

Comment:
will update this with something similar to [traffic-switch-on-chain-quote-provider.ts](https://github.com/Uniswap/routing-api/blob/267ec26f90eee67a0610a1bbd6181587b450105d/lib/handlers/quote/provider-migration/v3/traffic-switch-on-chain-quote-provider.ts)

Reply:
sampling will happen in a follow up PR